src/Goerli-DssSpell.t.sol

```diff
@@ -485,31 +488,56 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         // assertEq(vest.rxd(1), WAD);
     }
 
-    function testTeleportFW() public {
+     // RWA Tests
+
+    string RWA007_OLDDOC      = "QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY";
+    string RWA007_NEWDOC      = "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR";
+
+    string RWA008_OLDDOC      = "QmdfzY6p5EpkYMN8wcomF2a1GsJbhkPiRQVRYSPfS4NZtB";
+    string RWA008_NEWDOC      = "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy";
+
+    string RWA009_OLDDOC      = "QmQx3bMtjncka2jUsGwKu7ButuPJFn9yDEEvpg9xZ71ECh";
+    string RWA009_NEWDOC      = "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT";
+
+    function testRWA007DocChange() public {
```

Comment:
These tests can be bundled into one test named `testRWADocsChanges` (which I think is better, amd aligns better with other tests)

Reply:
It can't be done as `checkRWADocUpdate` in `*.t.base` casting the spell, so we need to call it in separate test cases

Reply:
ok, let's leave as is.